circle.yml

```diff
@@ -1247,7 +1257,7 @@ jobs:
 
   runner-integration-tests-firefox:
     <<: *defaults
-    resource_class: medium
+    resource_class: medium+
```

Comment:
The resources were bumped for the Firefox tests due to increased memory consumption. https://github.com/cypress-io/cypress/issues/21319 was logged to investigate further.